src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersistenceConfiguration.cs

```diff
@@ -52,6 +33,6 @@ public class SagaPersistenceConfiguration
         /// <summary>
         /// How long the client should attempt to acquire a lock when using pessimistic locking before giving up.
         /// </summary>
```

Comment:
Now I realize we could add a remark section to the XML documentation that mentions the default value here.